.pre-commit-config.yaml

```diff
@@ -1,11 +1,19 @@
 repos:
-    - repo: https://github.com/PyCQA/isort
-      rev: 5.12.0
+    - repo: https://github.com/charliermarsh/ruff-pre-commit
+      rev: v0.0.254
       hooks:
-          - id: isort
+        - id: ruff
+    - repo: https://github.com/Lucas-C/pre-commit-hooks
+      rev: v1.4.2
+      hooks:
+        - id: forbid-tabs
+          exclude: Makefile|docs/Makefile|\.bat$
+        - id: remove-tabs
+          exclude: Makefile|docs/Makefile|\.bat$
```

Comment:
Oo, I didn't know this was an option. Do we need both?

Reply:
good point, we probably don't need `forbid-tabs` if we also have `remove-tabs`?

Reply:
I've removed `forbid-tabs`

Reply:
Nice